### PR TITLE
0.6.8 release

### PR DESCRIPTION
**Pull request by korken89**

### CODE REVIEW

**Comment by korken89** [comment]
@thejpster  I think you have to write bors r+ without the @ :)

**Comment by korken89** [comment]
I pulled in the fix for the FPU flag so we don't have to do an extra release.

r? @rust-embedded/cortex-m

**Comment by korken89** [comment]
Hmm, `compiletest-rs` seems to have broken with the latest nightly.

**Comment by korken89** [comment]
Discussion here: https://github.com/laumann/compiletest-rs/issues/166

**Comment by thejpster** [comment]
How's it going?

**Comment by therealprof** [comment]
bors try

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.6.8] - 2019-03-18
```
Comment (therealprof) [inline CHANGELOG.md]:
```suggestion
## [v0.6.8] - 2019-04-02
```

**Pull request merged**